playground/compiler-express/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@playground/compiler-express",
+  "version": "0.0.0",
```

Comment:
Changesets requires everything in the monorepo to be versioned - even if it's not going to publish them